python/lib/dependabot/python/update_checker/requirements_updater.rb

```diff
@@ -278,14 +278,14 @@ def update_requirements_range(requirement_strings)
             requirement_strings.map { |r| requirement_class.new(r) }
 
           updated_requirement_strings = ruby_requirements.flat_map do |r|
-            next r.to_s if r.satisfied_by?(latest_resolvable_version)
+            next r.to_s if r.satisfied_by?(latest_resolvable_version, Requirement::BUMP_VERSIONS_OPS)
 
             case op = r.requirements.first.first
             when "<"
-              "<" + update_greatest_version(r.requirements.first.last, latest_resolvable_version)
-            when "<="
-              "<=" + latest_resolvable_version.to_s
-            when "!=", ">", ">="
+              "#{op}#{update_greatest_version(r.requirements.first.last, latest_resolvable_version)}"
+            when "<=", ">="
+              "#{op}#{latest_resolvable_version}"
+            when "!=", ">"
               raise UnfixableRequirement
```

Comment:
@kbukum1 Could we also add test for the `UnfixableRequirement` case.

Reply:
Checking it

Reply:
I added tests for Unfixable and actually since tests were same for each requirement file, requirement.txt, setup.py and setup.cfg. I shared tests for root context  to reduce duplication. Please let me know if you think that is a problem.